hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/upgrade/UpgradeTask.java

```diff
@@ -383,7 +384,7 @@ public static class UpgradeContainerResult {
     private final ContainerData originContainerData;
     private ContainerData newContainerData;
     private long totalRow = 0L;
-    private final long startTimeMs = System.currentTimeMillis();
+    private final long startTimeMs = Time.monotonicNow();
     private long endTimeMs = 0L;
```

Comment:
`endTimeMs` is still using `ystem.currentTimeMillis()`.

Reply:
same here

https://github.com/apache/ozone/blob/86b75054cf84fd90795135c267eabc516dd71518/hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/upgrade/UpgradeTask.java#L386-L387